overlord/servicestate/quota_handlers.go

```diff
@@ -203,6 +203,9 @@ func addRestartServicesTasks(st *state.State, queueTask func(task *state.Task),
 		for _, svc := range services {
 			names = append(names, svc.Name)
 		}
+		sort.Slice(names, func(i, j int) bool {
```

Comment:
I think `sort.Strings(names)` would work here too

Reply:
Ah nice, totally forgot about that one!